setup.py

```diff
@@ -67,6 +67,7 @@
     'pytest-django==3.1.2',
     'pytest-xdist==1.20.0',
     'tox==2.8.2',
+    'python-memcached==1.59'
```

Comment:
We should not be adding this to the setup.py dependencies. Projects should be able to choose which cache backend to use.

Reply:
This is similar to Postgresql client - it's by default included into the tests requirements, although users may need to choose another one, but we use it to run builds by default since it is frequently used. Memcached is also recommended cache backend and is similarly important part of stack.